docs/DeveloperGuide.md

```diff
@@ -8,22 +8,107 @@
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### _Designs_
+### 1. Calories Management Mode
+![Commands for managing calories](./diagrams/CaloriesManagement.png)
+   * when `caloriesManagement` is executed in Parser, different actions will be carried out based on the commands.
+   * if `eat`, store the name and calories of the input food
+   * if `view`, display all the foods consumed
+
+### 1. View all groceries added
+   * First create a method in "Grocery" class that prints the grocery in a preferred format.\
+     e.g., NAME, AMOUNT, EXPIRATION, PRICE.
+   * Then create a method in "GroceryList" class that prints all the groceries in the list.
+
+
+### 2. List the groceries by price in descending order
+   * First, create a field in "Grocery" class that stores the cost of a grocery.
+   * When adding a grocery, prompt the user to enter the cost.
+   * Format the cost into 2 decimal places, remove the dollar sign and store it as a string.
+   * Second, the grocery's cost accordingly before adding it into the list.
```

Comment:
I think there seems to be a grammatical error here which makes the meaning difficult to understand.